provisioner/powershell/provisioner.go

```diff
@@ -98,13 +98,15 @@ type Provisioner struct {
 
 func (p *Provisioner) defaultExecuteCommand() string {
 	baseCmd := `& { if (Test-Path variable:global:ProgressPreference)` +
-		`{set-variable -name variable:global:ProgressPreference -value 'SilentlyContinue'};`
+		`{set-variable -name variable:global:ProgressPreference -value 'SilentlyContinue'};` +
+		`{set-variable -name variable:global:ErrorActionPreference -value 'Continue'};` +
+		`%s. {{.Vars}};try { & '{{.Path}}' } catch { Write-Error $Error[0]; exit 1 }; if ($LastExitCode) { exit $LastExitCode }}`
 
+	var debugLine string
 	if p.config.DebugMode != 0 {
-		baseCmd += fmt.Sprintf(`Set-PsDebug -Trace %d;`, p.config.DebugMode)
+		debugLine = fmt.Sprintf(`Set-PsDebug -Trace %d;`, p.config.DebugMode)
 	}
-
-	baseCmd += `. {{.Vars}}; &'{{.Path}}'; exit $LastExitCode }`
+	baseCmd = fmt.Sprintf(baseCmd, debugLine)
```

Comment:
this doesn't look like a valid fmt.Sprintf string to me.

https://play.golang.org/p/tvHz_A_xWqL

Reply:
Yeah, there was a hard to see string interpolation verb at the beginning of the string that was hard to see. I reformatted it back to how it was as it was clearer.

Reply:
oh yeah I totally missed that, sorry.